test/suites/fdleak.sh

```diff
@@ -40,7 +40,7 @@ test_fdleak() {
 
   bad=0
   # shellcheck disable=SC2015
-  [ ${leakedfds} -gt 5 ] && bad=1 || true
+  [ "${leakedfds}" -gt 5 ] && bad=1 || true
```

Comment:
This should be a separate commit please.

Reply:
done